cibuildwheel/windows.py

```diff
@@ -1,91 +1,132 @@
 from __future__ import print_function
 import os, tempfile, subprocess, shutil, sys
 from collections import namedtuple
+from contextlib import contextmanager
 from glob import glob
 
 try:
     from shlex import quote as shlex_quote
 except ImportError:
     from pipes import quote as shlex_quote
 
+try:
+    from urllib.request import urlopen
+except ImportError:
+    from urllib2 import urlopen
+
 from .util import prepare_command, get_build_verbosity_extra_flags
+if hasattr(sys, 'getwindowsversion'):
```

Comment:
Why? I'm not sure I understand why you're testing for this? Don't we already know we're on Windows? And if not, won't `pep514tools` just fail?

Reply:
`windows.py` is imported by `__main__.py` unconditionnaly. Importing `pep514tools` fails with an import error if not protected because it uses modules only available on windows.

Reply:
OK, great!